requirements-dev.txt

```diff
@@ -3,6 +3,7 @@ flake8-import-order==0.18.1
 flake8==3.7.7
 groupy>=0.5.0
 mock==2.0.0
+more-itertools==5.0.0
```

Comment:
What does this do?

Reply:
Fixes Python 2.7, which the most recent version of this package is now incompatible with.

Reply:
Oh, you're saying that some other dependency listed in requirements-dev.txt pulls in more-itertools without a version cap, and the latest version is Python-2.7-incompatible?

What's pulling in a newer version? I just recreated my virtualenv and still got 5.0.0, and even `pip install -U more-itertools` says that 5.0.0 is the latest version, so it seems like something is correctly figuring this out. (My Python 3 virtualenv pulls in 7.0.0.)

I'm just reluctant to add dependencies that aren't actually dependencies, since it's easy to lose track of why they're there.

Reply:
Good point — my first guess, if it's working for you elsewhere, is that Docker is using the version of pip back from 16.04 which maybe isn't careful about Python version compatibility? I'll check!

Reply:
I can't reproduce the problem. It must have been an artifact of the order in which I was doing things in an earlier version of this Dockerfile!